spec/models/file_attachment/file_revision_spec.rb

```diff
@@ -2,23 +2,39 @@
 
 RSpec.describe FileAttachment::FileRevision do
   describe "#ensure_assets" do
-    it "doesn't change an asset that already exists" do
+    it "doesn't change the assets when they already exist" do
       file_revision = build(:file_attachment_file_revision)
-      file_asset = file_revision.file_asset
+      assets = file_revision.assets
```

Comment:
Just a minor thing, but as this is an unevaluated collection, I think this just checks `true == true`?

Reply:
I was a little bit confused by this comment, it did lead me to fix an issue and hopefully it's the same one you're referencing as they sound rather similar.

By comparing an instance of CollectionProxy in the expectation we weren't actually testing if anything changed, because even if the contents of the CollectionProxy change they'll still be the same object. 

I've resolved this by converting to an array.

Reply:
Sorry, I put it on the wrong line, which wasn't helpful :-|.